frontend/webapp/components/side.menu/menu/menu.tsx

```diff
@@ -76,15 +78,14 @@ export function Menu() {
   function renderContactUsButton() {
     return (
       <ContactUsWrapper>
-        <ContactUsButton expand={isHovered} />
+        <ContactUsButton expand={isExpanded} />
       </ContactUsWrapper>
     );
   }
 
   return (
     <MenuContainer
-      onMouseEnter={() => setIsHovered(true)}
-      onMouseLeave={() => setIsHovered(false)}
+      className={isExpanded ? 'isExpanded' : ''}
```

Comment:
please see instruction about passing props from styled-component docs: https://styled-components.com/docs/basics#passed-props

Reply:
changed the way prop is passed to the style.div according to docs